assets/javascripts/job_templates.js

```diff
@@ -394,7 +394,14 @@ function submitTemplateEditor(button) {
     // Reset to the minimum viable YAML if empty
     var template = editor.doc.getValue();
     if (template === '') {
-        template = "products: {}\nscenarios: {}";
+        template = "products: {}\nscenarios: {}\n";
```

Comment:
That seems redundant given the next block?

Reply:
I changed the next `if` to `else if`